lib/Onnxifi/HostManagerOnnxifi.cpp

```diff
@@ -104,6 +106,8 @@ HostManagerGraph::run(std::unique_ptr<ExecutionContext> ctx,
                  res->size() * res->getType().getElementSize());
         }
 
+        setTraceEvents(traceEvents, *ctx->getTraceContext());
```

Comment:
getTraceContext can return nullptr if the initial context does not set a TraceContext.

Reply:
yeah good point, will check that first